common/v2/services/ApiService/Gas/gasPriceFunctions.tsx

```diff
@@ -1,7 +1,7 @@
-import { gasPriceDefaults } from 'config/data';
-import { getNetworkByName } from 'v2/libs/networks/networks';
-import { GasEstimates, fetchGasEstimates } from 'v2/api/gas';
-import { Network } from '../Network/types';
+import { gasPriceDefaults } from 'v2/config/data';
+import { GasEstimates, Network } from 'v2/types';
+import { getNetworkByName } from '../../Store';
```

Comment:
Can be simplified/rewritten as `v2/services`.

Reply:
Yes this one is nasty.
I've been encountering issues with circular dependencies during babel imports. Very frustrating since they seem random and at times an import which is recognised by IDE appears as `undefined` after babel transforms.
changed to `v2/services/Store`